src/main/java/lib/choreolib/ChoreoTrajectoryState.java

```diff
@@ -51,4 +53,15 @@ public double[] asArray() {
       timestamp, x, y, heading, velocityX, velocityY, angularVelocity,
     };
   }
+
+  public ChoreoTrajectoryState flipped() {
+    return ChoreoTrajectoryState.builder()
+        .x(FIELD_WIDTH_METERS - this.x)
+        .y(this.y)
+        .heading(((this.heading + (Math.PI / 2)) * -1) - (Math.PI / 2))

```

Comment:
This math is a little overcomplicated. Can't this just be `Math.PI-this.heading`?